checkstyle-tester/projects-to-test-on.properties

```diff
@@ -8,12 +8,7 @@
 #sevntu-checkstyle|git|https://github.com/sevntu-checkstyle/sevntu.checkstyle|master||
 #checkstyle-sonar|git|https://github.com/checkstyle/sonar-checkstyle|master||
 
-#openjdk7|hg|http://hg.openjdk.java.net/jdk7/jdk7/jdk/|||
-#openjdk8|hg|http://hg.openjdk.java.net/jdk8/jdk8/jdk/|default|
-#All details at checkstyle/checkstyle#3033: ModalDialogActivationTest till JDK-8166015 ; 'jxc/8073519/**' not compilable by design ; ', jhsdb/**' - checkstyle do not support unicode identifiers
-#openjdk9|hg|http://hg.openjdk.java.net/jdk9/jdk9/jdk/|default|**/test/java/awt/Focus/ModalDialogActivationTest/ModalDialogActivationTest.java,**/test/javax/xml/bind/jxc/8073519/**,**/test/sun/tools/jhsdb/**
-#openjdk10|hg|http://hg.openjdk.java.net/jdk10/jdk10/jdk/|default|**/test/javax/xml/bind/jxc/8073519/**,**/test/sun/tools/jhsdb/**
-#openjdk11|git|https://github.com/openjdk/jdk11u|master||
+#openjdk10|git|https://github.com/AdoptOpenJDK/openjdk-jdk10u|dev|**/test/javax/xml/bind/jxc/8073519/**,**/test/sun/tools/jhsdb/**
```

Comment:
Why is this 10 and not 11?
https://github.com/checkstyle/contribution/issues/524#issue-743347042

Reply:
There are files that cause an exception in Checkstyle in openjdk11; the list is nearly identical to the one found at https://github.com/checkstyle/checkstyle/blob/master/.ci/openjdk14-excluded.files.

Reply:
ok.

Reply:
This way, we can simply tell a contributor to "uncomment all projects" when creating diff reports, with no caveats.

Reply:
Exceptions shouldn't matter to regression. Regression was built to report changes to exceptions. Like if a new exception came in or an old one went away. The only quirk to this was stack trace line numbers change and some don't want them to be displayed, regardless.